client/components/segmentation-survey/hooks/use-segmentation-survey-navigation.ts

```diff
@@ -79,6 +81,11 @@ const useSegmentationSurveyNavigation = ( {
 		recordContinueEvent( currentQuestion, answers );
 
 		await onContinue?.( currentQuestion );
+
+		if ( skipNextNavigation?.( currentQuestion.key, answers?.[ currentQuestion.key ] ) ) {
```

Comment:
cc. @phcp incase you have any thoughts on these changes inside the segmentation survey.

Reply:
Thanks for pinging, it looks good to me. 😄